DevCycle.SDK.Server.Local.Example/DevCycle.SDK.Server.Local.Example.csproj

```diff
@@ -18,4 +18,9 @@
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2-beta2" />
     <PackageReference Include="RestSharp" Version="108.0.2-alpha.0.6" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\DevCycle.SDK.Server.Common\DevCycle.SDK.Server.Common.csproj" />
+    <ProjectReference Include="..\DevCycle.SDK.Server.Local\DevCycle.SDK.Server.Local.csproj" />
+  </ItemGroup>
```

Comment:
Should this be here?

Reply:
the external SDKs are referenced above.

Reply:
Yes - local reference to ensure latest pull.

Reply:
but is that what we want with the examples? don't we kinda want the examples to be portable so someone could use them outside of this project. Really all these example changes should be in a separate PR.

Reply:
The way project references work in dotnet is that if it can find a local reference (the path references) it uses that first. So if you're using this in context of the SDK repo - it will pull from the latest repo. 
If it can't find it - it searches for other public depends that satisfy the same namespace. (the released versions above)

This maintains portability, and ensures latest functionality checks.

Reply:
If we want to remove this in the future - it's not a breaking change at all